.changeset/yellow-debt-v2-confidence-calibration.md

```
@@ -0,0 +1,40 @@
+---
+"yellow-debt": minor
+---
+
+Scanner output schema v2.0 — failure_scenario field, audit-synthesizer
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)